ui/app/routes/vault/cluster/secrets/backend/configuration/edit.ts

```diff
@@ -55,20 +56,20 @@ export default class SecretsBackendConfigurationEdit extends Route {
     // generate the model based on the engine type.
     // and pre-set model with type and backend e.g. {type: ssh, id: ssh-123}
     const model: Record<string, unknown> = { type, id: backend };
-    for (const adapterPath of CONFIG_ADAPTERS_PATHS[type] as string[]) {
-      // create a key that corresponds with the configs model order
-      // ex: adapterPath = ssh/ca-config, convert to: first-model so that you can pass to component @model={{this.model.first-model}}
-      const standardizedKey = this.standardizedModelName(type, adapterPath);
+    for (const modelName of MOUNT_CONFIG_MODEL_NAMES[type] as string[]) {
```

Comment:
Thanks for the rename, this is easier to parse (but also grrr Ember data, I will not miss you when you're gone)

Reply:
agree!